Packs/CloudIncidentResponse/Playbooks/playbook-Cloud_Data_Exfiltration_Response.yml

```diff
@@ -0,0 +1,1179 @@
+    type: title
+    task:
+      id: 7edb2290-6dfc-4c91-899f-4a73b47ca242
+      version: -1
+      name: Check IP Prevelance
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "43"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": 320
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "28":
+    id: "28"
+    taskid: ef365d2c-9f1e-4d23-8c0d-370585b85f13
+    type: title
+    task:
+      id: ef365d2c-9f1e-4d23-8c0d-370585b85f13
+      version: -1
+      name: Enrichment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "6"
+      - "23"
+      - "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": 180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: 0bd1ea2c-e85f-4172-8763-772bfa9a3057
+    type: title
+    task:
+      id: 0bd1ea2c-e85f-4172-8763-772bfa9a3057
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1260,
+          "y": 2380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: b2a335ad-2aaf-4e7a-8ccf-252137bebdd2
+    type: regular
+    task:
+      id: b2a335ad-2aaf-4e7a-8ccf-252137bebdd2
+      version: -1
+      name: Close Alert
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "30"
+    scriptarguments:
+      closeReason:
+        simple: True Positive - Handled by automation.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": 2210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 9c4eb7f8-d21f-4ac7-83bc-167244ce5cb7
+    type: condition
+    task:
+      id: 9c4eb7f8-d21f-4ac7-83bc-167244ce5cb7
+      version: -1
+      name: Found malicious evidence based on enrichment data
```

Comment:
Why aren't we using the prevalence results?

Reply:
its not enough to set the alert to malicious in my opinion

Reply:
It can be combined with other enrichments